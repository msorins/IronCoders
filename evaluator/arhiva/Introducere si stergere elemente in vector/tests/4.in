7
1 532 21 56 21 5 0
5
0 2
0 1
1 3 532
1 3 4
0 4