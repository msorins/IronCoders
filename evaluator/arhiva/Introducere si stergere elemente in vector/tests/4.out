21 56 4 21 5 0 